### PR TITLE
Code cleanup

### LINKED ISSUE #150: Improvements in code and suggestions
https://www.loom.com/share/4928c733a3014eb19e614c2d7eab0cf4

@AbhayVAshokan _a

Please take this up and assign to your tech lead once done.

cc: @labeebklatif

### PR DESCRIPTION
**Pull request by labeebklatif**
Fixes #150 

@karthiknmenon _a Please review the changes

### CODE REVIEW

File: lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js [hunk 1]
```diff
@@ -33,19 +34,19 @@ export default {
   }) => {
     let commandItems = [
       {
+        optionName: ADD_ON_OPTIONS.FONT_SIZE,
         title: "Paragraph",
         description: "Add a plain text block",
```
Comment (karthiknmenon) [inline lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js:39]: Can we move the title and descriptions to constants as well? 
Also, please add periods to the end of each sentence in descriptions.

File: lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js [hunk 2]
```diff
@@ -19,8 +15,13 @@ import {
   Email,
   Video,
 } from "@bigbinary/neeto-icons";
```
Comment (karthiknmenon) [inline lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js:17]: Let's update neetoIcons to the latest version.

File: lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js [hunk 3]
```diff
@@ -1,10 +1,6 @@
-import { Extension } from "@tiptap/core";
-import CommandsList from "./CommandsList";
-import { PluginKey } from "prosemirror-state";
-import Suggestion from "@tiptap/suggestion";
 import tippy from "tippy.js";
-import { ReactRenderer } from "@tiptap/react";
 import isEmpty from "lodash.isempty";
```
Comment (karthiknmenon) [inline lib/components/Editor/CustomExtensions/SlashCommands/ExtensionConfig.js:2]: Can we use ramda instead of lodash at all places in the repo?

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@labeebklatif _a Few minor suggestions. Please check.

**Comment by labeebklatif** [comment]
@karthiknmenon _A suggested changes done. Please have a look

**Comment by labeebklatif** [comment]
@labeebklatif _a

**Comment by labeebklatif** [comment]
@karthiknmenon _A suggested changes done. Please have a look

**Review by karthiknmenon — APPROVED** [review]

**Pull request merged**